Tools/NTupleReader.h

```diff
@@ -664,12 +664,12 @@ class NTupleReader
         else if(convertHackActive_ && intuple) //else check if it is a vector<float> or vector<double>
         {
             //hack to get vector<double> as vector<float>, requires DuplicateFDVector() to be run
-            char typen;
+            std::string typen;
```

Comment:
this is fundamentally different than a char??

Reply:
Yeah but the compiler wants a default. It was easier to just make it a string.

Reply:
easier than doing `char typen = NULL;'?

Reply:
NULL is for a pointer. If you use that it will convert it to '\0' and give a warning. I just updated it with '\0' instead.